### ISSUE TITLE
'... shall be [fined, imprisoned, or both]'

### ISSUE THREAD

**Issue by bwaldon** [nert-nlp/cgel#142]
Should we treat this as ternary-branching coordination? If so, the first two coordinates are presumably VP... how about "both"?

**Comment by nschneid** [comment]
Is "both" elliptical for "both fined and imprisoned", and if so, is it a VP with "both" functioning as fused Marker-Head? Or should we just say that anaphora can lead to coordination of unlike categories?

**Comment by BrettRey** [comment]
Yes, I think _both_ is a fused Marker-Head.

3.2.4 Marker-Head
We have two cases of a phrase in Marker-Head function: The case of etc. (see §5.4.3)
and the case of elliptical stranding of to (see §6.1.1).

**Comment by nschneid** [comment]
@BrettRey is this what you mean? It gives a Coordination headed by a DP which is anomalous, but I can't think of a better solution.

![image](https://github.com/user-attachments/assets/dc1a5faf-0d8a-4d52-8013-ef859d4d2e40)

**Comment by BrettRey** [comment]
yes. Does that make sense?

**Comment by nschneid** [comment]
There was some discussion about whether "both" is a fused Det-Head NP, roughly short for "both of these" or "both things", which would be similar to analyzing "etc." as a fused Marker-Head NP parallel to "and others".

Intuitively, though, it feels like "both" is used here like the Marker in correlative coordination.

Notably, "all" cannot appear in correlative coordination, and also is strange in this construction:

- ??The defendant shall be fined, imprisoned, censured, or all.

"All of the above" is the ordinary way to express this, and is a clear NP.

The other correlative coordinations are either...or and neither...nor. "Neither" also works in the right context:

- You may be fined, imprisoned, or neither.

It's hard for me to think of a case where "either" makes sense semantically since "or" is already typically disjunctive.

**Comment by nschneid** [comment]
(A different construction with "neither":

- You like ice skating, and so do I.
- You don't like ice skating, and neither do I.

I don't think this bears on our analysis of the above.)

**Comment by nschneid** [comment]
TL;DR of our analysis:

- The last coordinate of a VP-Coordination may be an anaphoric NP ("etc.", "no others", "all of the above", "both of these", "neither one", ...).
- In a list of 2+ things followed by "or both/neither", we treat it as ellipsis of an embedded coordination, of which *both*/*neither* is a fused marker-head. Unless punctuation clearly signals it is a supplement, we treat it as a final coordinate.
